luxtronik/datatypes.py

```diff
@@ -504,10 +512,23 @@ class HeatpumpCode(SelectionBase):
         69: "MSW 6S",
         70: "MSW 8S",
         71: "MSW 10S",
-        72: "MSW 13S",
+        72: "MSW 12S",
```

Comment:
Are we sure about this change? Where was this originally coming from?

Reply:
The `13S` seems to be present in all open source projects reading data from the luxtronik controller. But the luxtronik controller itself would present these heat pumps as `12S`.  Google doesn't find anything related to these strings...

Reply:
I did a quick check against the `appl` binary of the heatpump and there I find only `12S` but n `13S`:

```sh
strings firmware/appl | grep 12S$
MSW_3_12S
MSW12S
MSW3-12S
MSW_3_12S
```

```sh
strings firmware/appl | grep 13S$
```

So I guess `12S` is right